tests/v3/conftest.py

```diff
@@ -77,6 +78,14 @@ async def store(request: pytest.FixtureRequest, tmpdir: LEGACY_PATH) -> Store:
     return await parse_store(param, str(tmpdir))
 
 
+@pytest.fixture(params=["local", "memory", "zip"])
+def sync_store(request: pytest.FixtureRequest, tmpdir: LEGACY_PATH) -> Store:
```

Comment:
While this is being touched, I think it's worth replacing `tmpdir` with `tmp_path` which is the modern way to do temporary directories with pytest